templates/layouts/configs/_census-theme.html

```diff
@@ -0,0 +1,8 @@
+{% set page = page | setAttributes({
+  "theme": "census"
+}) %}
+
+{% set page = page.header | setAttributes({
+  "titleLogo": "/img/census-logo-" + language_code + ".svg",
+  "titleLogoAlt": "Census 20201" if language_code == "en" else "Cyfrifiad 2021"
```

Comment:
I'm not gonna bother with 20201.
```suggestion
  "titleLogoAlt": "Census 2021" if language_code == "en" else "Cyfrifiad 2021"
```

Reply:
Amazing typo there! Yeah I guess we leave it as this for now